client/state/ui/actions.js

```diff
@@ -1,7 +1,11 @@
 /**
  * Internal dependencies
  */
-import { SET_SELECTED_SITE, SET_SECTION } from 'state/action-types';
+import {
+	SET_SELECTED_SITE,
+	SET_SECTION,
+	USER_SET_CURRENT
```

Comment:
Added a section to data document in #1378 describing actions, including desire to suffix verbs (aff948ec88f67dd58ddc052db75ac5199355488b).

I'll create a pull request for changes after merging this, as I expect there'd be merge conflicts otherwise.